t/e2e/whisper/whisper_test.go

```diff
@@ -125,11 +134,22 @@ func (s *WhisperTestSuite) TestLogout() {
 
 	// make sure that identity doesn't exist (yet) in Whisper
 	s.False(whisperService.HasKeyPair(pubKey), "identity already present in whisper")
+
+	// select account
 	s.NoError(s.Backend.SelectAccount(address, TestConfig.Account1.Password))
-	s.True(whisperService.HasKeyPair(pubKey), "identity not injected into whisper")
+
+	// Get the chat account
+	selectedChatAccount, err := s.Backend.AccountManager().SelectedChatAccount()
+	s.NoError(err)
+
+	// Chat account public key should be the same key received when creating the account
```

Comment:
Keys being the same has nothing to do with `Logout` logic.

Reply:
@adambabik for now `CreateAccount` returns only the wallet key. the first part of the test is testing that the returned key is not in whisper, so I wanted to be sure that the key we are testing is the same the we are using for chat. It will be removed in the next steps when we'll introduce a different key, so that part will fail and force us to update the test. 
If you prefer I can remove it

Reply:
I saw the other comment, so if we create a different test to check the 2 keys, it makes sense to remove from here!